doc/Configuration.md

```diff
@@ -0,0 +1,84 @@
+
+#### Samples Sheet
+
+To make it simple to specify your fastq files and their locations, along with associated sample names and replicate numbers, we provide a CSV file (`samples.csv`) which can be modified in a spreadsheet editor such as Microsoft Excel or LibreOffice Calc. [More info](#samples-sheet-details).
+
+#### Features Sheet
+
+Small RNAs can often be classified by sequence characteristics, such as length, strandedness, and 5' nucleotide. We provide a Features Sheet (`features.csv`) in which you can define selection rules to more accurately capture counts for the small RNAs of interest. [More info](#features-sheet-details).
+
+#### Plot Stylesheet
+
+Plot styles can be optionally overridden using a matplotlibrc stylesheet. [More info](#plot-stylesheet-details).
+
+## Run Config Details
+
+### The processed Run Config
+When the pipeline starts up, tinyRNA will process the Run Config based on the contents of it and your other configuration files, and the processed copy will be saved to your run directory. The processed configuration is what ultimately determines the behavior of the workflow. This provides auto-documentation for all of your pipeline runs.
+
+## Paths File Details
+
+### Building Bowtie Indexes
+If you don't have bowtie indexes already built for your reference genome, tinyRNA can build them for you at the beginning of an end-to-end run and reuse them on subsequent runs with the same Paths File.
+
+To build bowtie indexes:
+1. Open your Run Config in a text editor and find the `run_bowtie_build` key. Set its value to `true` and save it.
+2. Open your Paths File in a text editor and find the `reference_genome_files` key. Add your reference genome file(s) under this key, one per line with a `- ` in front.
+3. Find the `ebwt` key and delete its value.
+4. Execute an end-to-end pipeline run.
+
+Once your indexes have been built, your Paths File will be modified such that `ebwt` points to their location (prefix) within your Run Directory. This means that indexes will not be unnecessarily rebuilt on subsequent runs as long as the same Paths File is used. If you need them rebuilt, simply repeat steps 3 and 4 above.
+
+## Samples Sheet Details
+
+### Assigning the Control Group
+Assigning the control group allows the proper DGE comparisons to be made and plotted. The Control column is where you'll make this indication by writing `true` on any corresponding row. Regardless of the number of replicates in each group, only one associated row needs to have this indication. Do not write `false` or anything else for the other groups; this column should only be used to indicate the affirmative.
```

Comment:
In an earlier version that was the case, but it has since been fixed. Only the word 'true' is matched regardless of case. The last sentence can be removed.